tests/cli/test_login.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018-2021 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test ``login`` command."""
+
+from renku.cli import cli
+from renku.core.commands.login import read_renku_token
+
+ENDPOINT = "renku.deployment.ch"
+USER_CODE = "valid_user_code"
+
+
+def test_login(runner, client, mock_login):
+    """Test login command."""
+    result = runner.invoke(cli, ["login", ENDPOINT], input=USER_CODE)
+
+    assert 0 == result.exit_code
+    assert "jwt-token" == read_renku_token(client, ENDPOINT)
```

Comment:
I think `mock_login` should yield the token strings ('jwt-token', 'other-token') along with the mock, then we can compare to that here (not have the string defined in two places)